src/condor_utils/file_transfer.cpp

```diff
@@ -4509,16 +4553,10 @@ FileTransfer::computeFileList(
 				// While (* total_bytes_ptr) and numFiles should both be 0
 				// at this point, we should probably be explicit.
 				filesize_t logTCPStats = 0;
-				return ExitDoUpload( & logTCPStats,
-					/* num files */ 0,
-					s, saved_priv, protocolState.socket_default_crypto,
-					/* upload success */ false,
-					/* do upload ACK (required to put job on hold) */ true,
-					/* do download ACK */ false,
-					/* try again */ false,
-					FILETRANSFER_HOLD_CODE::UploadFileError,
-					/* hold subcode */ 3,
-					errorMessage.c_str(), __LINE__ );
+				UploadExitInfo xfer_info;
+				xfer_info.setError(errorMessage, FILETRANSFER_HOLD_CODE::UploadFileError, 3);
+				xfer_info.doAck(TransferAck::UPLOAD).line(__LINE__);
```

Comment:
What does you sense of style say about
```suggestion
				xfer_info.setError(errorMessage, FILETRANSFER_HOLD_CODE::UploadFileError, 3)
				         .info.doAck(TransferAck::UPLOAD)
				         .line(__LINE__);
```
?